docs/tests/test_fields.py

```diff
@@ -0,0 +1,340 @@
+from collections import deque
+from copy import deepcopy
+from . import validator, _load_fixture
+
+
+def test_wrong_type_fields_int():
+    form = _load_fixture('0010_wrong_type_field_int.json')
+    errors = sorted(validator.iter_errors(form), key=lambda e: e.path)
```

Comment:
This line occurs very often, have you considered a helper function to improve readability?

Reply:
now that you're mentioning it...
I'll see how I can improve this.

Reply:
mmmm... thinking about it, I'd be replacing one line of code by... one line of code & a function definition + one line of code... is it worth it?

Reply:
I was thinking this line is a bit "heavy" to read, and replacing it by a function call could make things clearer. But it definitely requires more code... Your call, the PR is approved :+1: